crates/compiler/can/src/module.rs

```diff
@@ -996,7 +996,7 @@ fn fix_values_captured_in_closure_expr(
             ..
         }
         | Dbg {
-            loc_condition,
+            loc_message: loc_condition,
```

Comment:
Consider doing the local rename here too

Reply:
I renamed it to `loc_condition` so that I wouldn't need a separate match statement. This enables it to be match with `Expect` which still uses the name `loc_condition`